package.json

```diff
@@ -151,19 +155,17 @@
     "commander": "^2.20.0",
     "concurrently": "^4.1.1",
     "cross-env": "^5.2.0",
-    "csstype": "^2.6.10",
+    "csstype": "^3.0.11",
```

Comment:
`csstype` is used by Paste packages for style prop declarations, as well as Emotion. This has been bumped by emotion and included a breaking change (exported type interfaces).

This is a version bump only.